src/qml/qgismobileapp.qml

```diff
@@ -2727,18 +2729,14 @@ ApplicationWindow {
     }
 
     MenuItem {
-      id: copyCoordinatesItem
-      text: qsTr( "Copy Coordinates" )
+      id: lockMapRotation
+      text: "Enable Map Rotation"
       height: 48
-      leftPadding: Theme.menuItemLeftPadding
+      leftPadding: Theme.menuItemCheckLeftPadding
       font: Theme.defaultFont
-      icon.source: Theme.getThemeVectorIcon( "ic_copy_black_24dp" )
-
-      onTriggered: {
-        var displayPoint = GeometryUtils.reprojectPoint(canvasMenu.point, mapCanvas.mapSettings.destinationCrs, projectInfo.coordinateDisplayCrs)
-        platformUtilities.copyTextToClipboard(StringUtils.pointInformation(displayPoint, projectInfo.coordinateDisplayCrs))
-        displayToast(qsTr('Coordinates copied to clipboard'));
-      }
+      checkable: true
+      checked: qfieldSettings.enableMapRotation
```

Comment:
The screenshot shows an alignment issue between icons and the checkbox. This should do it:

```suggestion
      checked: qfieldSettings.enableMapRotation
      indicator.height: 20
      indicator.width: 20
      indicator.implicitHeight: 24
      indicator.implicitWidth: 24
```

Reply:
ok thank you..
### result:   
![image](https://github.com/opengisch/QField/assets/36326627/05b7c5f4-8515-4025-9b7b-2d62755cf05f)